elements/rh-secondary-nav/rh-secondary-nav-lightdom.css

```diff
@@ -60,9 +61,9 @@ rh-secondary-nav > [slot="logo"] {
   max-width: var(--rh-secondary-nav-logo-max-width, 10em);
   text-decoration: none;
   font-size: var(--rh-font-size-body-text-md, 1rem);
-  letter-spacing: 0.01125em; /* WARNING: not a recognized token value */
-  line-height: 1.6875em; /* WARNING: not a recognized token value */
-  margin-inline-start: var(--rh-space-2xl, 32px);
+  letter-spacing: var(--rh-letter-spacing-body-text, 0.01125em);
```

Comment:
is that the correct value?

Reply:
The original value for letter spacing was taken from the XD file of 0.18px, so at 16px as base 0.01125em would be correct.    The CSS property `--rh-letter-spacing-body-text` however isn't the same as in the tokens project, I must have [misread it](https://github.com/RedHat-UX/red-hat-design-tokens/blob/main/tokens/font/letter-spacing.yml) when copying token/values as it is set to `0.0125rem`.  Will remove the token.

Reply:
Fixed in 9f012b6d97cbebb79034124e6e8fd8ce1b19143e